### DISCUSSION TITLE
OpenCore 0.7.6 for ThinkPad T460s Monterey

### DISCUSSION THREAD

**Discussion by webdeveloper1877** [simprecicchiani/ThinkPad-T460s-macOS-OpenCore#137] [General]
**Hello everyone, since the geniuses who worked on this project can no longer follow it constantly, I would like to post the opencore version 0.7.6 with all the updated kext, the exact model of thinkpad on which it was tested and fully functional with MacOs Monterey is...**

Model | Thinkpad T460s 20F90002**
-- | --
Processor | Core i5-6300U (2C, 2.4 / 3.0GHz, 3MB)
Graphics | Integrated Intel HD Graphics 520
Memory | 4GB Soldered + 16GB DIMM 2133MHz DDR4, dual-channel
Display | 14" Full HD (1920x1080) IPS, Touch 
Storage | SAMSUNG MZNLN512HMJP 512GB SSD
Ethernet | Intel Ethernet Connection I219-LM (Jacksonville)
WLAN + Bluetooth | 11ac+BT, Intel® Dual Band Wireless-AC 8265, 2x2 card
Camera | HD720p resolution, low light sensitive, fixed focus
Audio support | HD Audio, Realtek ALC3245 codec, stereo speakers 1Wx2, dual array microphone, combo audio/microphone jack
Keyboard | 6-row, spill-resistant, multimedia Fn keys, LED backlight
Battery | Front Li-Polymer 3-cell (23Wh) and rear Li-Ion 3-cell (26Wh), both Integrated

**download** 
[EFI ThinkPad T460s OC 0.7.6 Monterey](https://mega.nz/file/4hhR2aKK#WAmycUEHmcEGGUdbMbG7YBRWXC6CVf28ve5Y-R_e1Ng)

**Comment by simprecicchiani** [comment]
Thanks for the early update. I found some time to publish the changes, please lmk if something doesn't work.
